packages/shared/src/api/postsApi.ts

```diff
@@ -662,7 +662,7 @@ export function toPagedPostsData(
 
 export function toPostsData(
   channelId: string,
-  posts: ub.Posts | Record<string, ub.Reply>
+  posts: ub.Posts | ub.Writs | Record<string, ub.Reply>
```

Comment:
My memory was that ub.Posts was generic + could handle posts or writs -- is that not the case?